README.md

```diff
@@ -5,11 +5,11 @@
 
 ---
 
-... a 19 y/o developer / student from Austria interested in many things ranging from IT-sec to optimizing performance with algorithms & datastructures. I'm currently studying at [HTBLA Kaindorf](https://www.htl-kaindorf.at/) and working on several open-source projects in my free time. 
+... a 20 y/o developer from Austria interested in many things ranging from IT-sec to optimizing performance with algorithms & datastructures. I have graduated from [HTBLA Kaindorf](https://www.htl-kaindorf.at/) and I am currently a Cyber GWD at the Austrian Armed Forces, while also working on several open-source projects in my free time. 
```

Comment:
drop cyber gwd part - just write "data structures, plus I enjoy working on open-source projects in my free time."